platforms/common/include/px4_platform_common/board_common.h

```diff
@@ -76,6 +76,10 @@
 #  endif
 #endif
 
+#ifndef BOARD_SPI_RAMTRON_SPEED_MHZ
+#define BOARD_SPI_RAMTRON_SPEED_MHZ 10
```

Comment:
kconfig

Reply:
I tried to understand how the RAMTRON speed is set in the code:

1. It's currently hardcoded to start initialization at 10MHz and if it fails tries 9 times again down to 1MHz.
2. Once the initialization succeeds and the part is detected, the speed is taken from the table (Between 25 and 40MHz) and set to priv->speed.
3. This table speed is then simply overwritten and set to between 1 and 10MHz.

So RAMTRON has is always running at slower than max speed. 

I don't understand what the use case for *dynamically* determining the max speed? The RAMTRON is not hot pluggable so unless your PCBs differ in quality, the max speed should always be the same. 

I think it the speed should be config option in the `px4_mft_device_t` struct and defaulted to use the max device speed. But that seems to be a larger refactoring, so I now hacked it to start at 30MHz, since I spent my timebox on this already.

Reply:
I believe it was just desperation for the extremely rare case parameters were failing to restore in the field and we had nothing to go on.